packages/quantic/cypress/e2e/default-2/generatedAnswer/generated-answer.cypress.ts

```diff
@@ -359,6 +368,27 @@ describe('quantic-generated-answer', () => {
       });
     });
 
+    describe('when a custom value is provided to the fields to include in citations attribute', () => {
+      const streamId = crypto.randomUUID();
+      const customFields = 'foo,bar';
+
+      beforeEach(() => {
+        mockSearchWithGeneratedAnswer(streamId);
+        mockStreamResponse(streamId, genQaMessageTypePayload);
+        visitGeneratedAnswer({fieldsToIncludeInCitations: customFields});
+      });
+
+      it('should send a search query with the right fields to include in citations option as a parameter', () => {
```

Comment:
```suggestion
      it('should perform a search query with the right fields to include in citations option as a parameter', () => {
```

Just a suggestion for the test suite to be more consistent with the previous test (default) description